app/Http/Controllers/UserController.php

```diff
@@ -10,7 +10,7 @@
 
 class UserController extends Controller
 {
-    public function index(){
+    public function index() {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->